README.md

```diff
@@ -59,15 +45,59 @@ to use Kubernetes as a PaaS (Platform as a Service).
 
 ## Documentation
 
-Installation and user documentation is available at our main [docs.epinio.io](https://docs.epinio.io/) site.
+Detailed installation and user documentation is available at our main
+[docs.epinio.io](https://docs.epinio.io/) site.
+
+### Installation
+
+Leaving out DNS setup in the most trivial case the main installation boils down to
```

Comment:
Just a suggestion. Given that we are mentioning the DNS setup, I am not sure if it would make sense to list the main requirements for the setup on the documentation (Kubernetes Cluster, Ingress Controller, Cert Manager). If so, perhaps it could be rephrased perhaps as something like this:

_The basic boilerplate requires a Kubernetes cluster, an Ingress Controller and a Cert Manager as detailed in the documentation. Once this is in place, leaving out DNS setup in the most trivial case the main installation boils down to:_

Reply:
Will do

Reply:
Addressed, commit fd197d63907c94339028a7bc408548281a1f46da